vulcano/app/classes.py

```diff
@@ -103,13 +103,14 @@ def run(self, prompt=u'>> ', theme=MonokaiTheme, print_result=True, history_file
         :param bool print_result: If True, results from functions will be printed.
         """
         self.theme = theme
+        self.prompt = prompt
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Attribute 'prompt' defined outside __init__](https://app.codacy.com/manual/david-gp87/vulcano/pullRequest?prid=5712243)

Reply:
I'm happy merging this, this could potentially avoid having to type a whole command as I did on the example above to modify the prompt. But, could it be possible adding the prompt inside the __init__ definition and add a docstring specifying that it could be a string or a function? Also keep in mind that py27 people could be interested on having `u'>>'` as a default prompt.